engine/common/api_helpers/mixins.py

```diff
@@ -351,9 +354,19 @@ def get_attr_template(self, attr, alert_receive_channel, render_for=None):
                 return Response({"preview": e.fallback_message}, status.HTTP_200_OK)
         else:
             templated_attr = None
-        response = {"preview": templated_attr}
+        response = {"preview": templated_attr, "is_valid_json_dict": self.is_valid_json_dict(templated_attr)}
         return Response(response, status=status.HTTP_200_OK)
 
+    def is_valid_json_dict(self, json_str):
```

Comment:
nit: maybe a better name for this would be `is_valid_json_object`? https://en.wikipedia.org/wiki/JSON#Data_types

Reply:
Right, thanks